From 9c469ebdd84e00dd5817f4124a9da3c5913a028f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 13 May 2020 15:54:10 +0300
Subject: [PATCH] [SW-0] preboot: "unweak" the preboot_version()

The preboot_version() function is not used anywhere besides preboot.c
file and is not overloaded with another variant.
Remove "weak" from its definition and staticize it.

Change-Id: Ia8ea2f67f2e54babfaeb2aa3150a9428bc5187be
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index a4d5253..68efc44 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -120,7 +120,7 @@
 static const char *preboot_ver_str = "Preboot " VERSION_STR;
 static char preboot_btl_ver_str[VERSION_MAX_LEN] = "BTL version ";
 
-__attribute__((weak)) void preboot_version(void)
+static void preboot_version(void)
 {
 	unsigned long offset = (unsigned long) preboot_btl_ver_str;
 	u32 btlver = regmap_readl(mmPREBOOT_VER_OFFSET);
